### PR TITLE
Update jquery

### PR DESCRIPTION
**Pull request by Mte90**
Actually I updated to jquery 2.2.4 because to 3.2.1 I got few errors.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "__MSG_extName__",
+    "name": "HoverZoom+",
```
Comment (extesy) [inline manifest.json]: this is required for localization

Comment (Mte90) [inline manifest.json]: You are right, I fixed :-)

**Comment by Mte90** [comment]
Finally finished the porting to jquery 3.2.1

**Comment by Mte90** [comment]
Seems that on google plugin there are issue but I am working on it

**Comment by Mte90** [comment]
I tested also on chrome and works without problems now (the google plugin now use the right selector)

**Pull request merged**